lib/oli_web/live/common/stepper/stepper.ex

```diff
@@ -0,0 +1,86 @@
+defmodule OliWeb.Common.Stepper do
+  use Phoenix.LiveComponent
+
+  alias OliWeb.Common.Stepper.Step
+
+  @moduledoc """
+  Stepper Component
+  """
+
+  @empty_step %Step{
+    title: "",
+    description: "",
+    render_fn: &__MODULE__.render_unselected_step/1,
+    data: %{}
+  }
+
+  attr :id, :string, required: true
+  attr :steps, :list, required: true
+  attr :current_step, :integer, default: 0
+  attr :data, :map, default: %{}
+
+  def render(assigns) do
+    assigns = assign(assigns, steps: Enum.with_index(assigns.steps))
+
+    assigns =
+      assign(assigns,
+        selected_step:
+          Enum.find(assigns.steps, {@empty_step, 0}, fn {_step, index} ->
+            index == assigns.current_step
+          end)
+          |> elem(0)
+      )
+
+    ~H"""
+      <div id={@id} class="flex md:flex-row flex-col-reverse h-screen w-full">
+        <div class="bg-delivery-body-color-dark dark:bg-gray-900 h-3/5 w-full md:h-full md:w-3/5" />
+        <div class="bg-blue-700 h-2/5 w-full md:h-full md:w-2/5" />
+        <div class="flex md:flex-row flex-col-reverse absolute p-16 top-0 bottom-0 left-0 right-0 m-auto">
+          <div class="bg-white dark:bg-gray-800 w-full md:w-2/3 border border-gray-200 dark:border-gray-600 flex flex-col overflow-hidden">
+            <%= @selected_step.render_fn.(@data) %>
+            <div class="px-9 py-4 flex items-center justify-between border-gray-200 dark:border-gray-600 border-t">
+              <button class="torus-button secondary">Cancel</button>
```

Comment:
We can add a `phx-click` binding with the corresponding `handle_event` callback to make this button actionable